src/main/java/core/basesyntax/CircleSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;

```

Comment:
The import statement for `java.util.Random` is redundant since you're not using it directly in this class. You can remove it to keep the code clean.